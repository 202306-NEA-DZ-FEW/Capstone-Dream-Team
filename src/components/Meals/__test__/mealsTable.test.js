import renderer from "react-test-renderer";
import MealsTable from "../mealsTable";

const meals = [
    {
        name: "Sushi",
        id: 5,
        maxMeals: 34,
        imageUrl: "",
        price: 5,
    },
];

it("renders correctly", () => {
    const tree = renderer.create(<MealsTable meals={meals} />).toJSON();
    expect(tree).toMatchSnapshot();
});
