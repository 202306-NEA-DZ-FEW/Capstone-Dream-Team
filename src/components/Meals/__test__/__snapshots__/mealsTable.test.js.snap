// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="overflow-x-auto overflow-y-hidden rounded-md border border-gray-200 shadow-md mt-5"
>
  <table
    className="w-full text-center text-sm rtl:text-right"
  >
    <thead
      className="text-xs  uppercase bg-blue-300 bg-opacity-50 "
    >
      <tr>
        <th
          className="px-6 py-3"
          scope="col"
        >
          mealsPage.mealsTable.image
        </th>
        <th
          className="px-6 py-3"
          scope="col"
        >
          mealsPage.mealsTable.name
        </th>
        <th
          className="px-6 py-3"
          scope="col"
        >
          mealsPage.mealsTable.quantity
        </th>
        <th
          className="px-6 py-3"
          scope="col"
        >
          mealsPage.mealsTable.price
        </th>
        <th
          className="px-6 py-3"
          scope="col"
        >
           
        </th>
      </tr>
    </thead>
    <tbody
      className=""
    >
      <tr
        className="bg-white border-b hover:bg-gray-100"
      >
        <td
          className="py-4"
        >
          <div
            className="flex justify-center"
          >
            <img
              alt="Sushi"
              className="w-20 md:w-20 max-w-full max-h-full"
              src=""
            />
          </div>
        </td>
        <td
          className="px-4 py-4 font-semibold text-gray-900 dark:text-white"
        >
          Sushi
        </td>
        <td
          className="px-6 py-4 font-semibold text-gray-900 dark:text-white"
        >
          34
        </td>
        <td
          className="px-6 py-4 font-semibold text-gray-900 dark:text-white"
        >
          $
          5
        </td>
        <td
          className="px-6 py-4"
        >
          <div
            className="flex justify-end gap-2"
          >
            <button
              className="text-red-700"
              onClick={[Function]}
            >
              <svg
                className="h-5 w-5"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M14.74 9l-.346 9m-4.788 0L9.26 9m9.968-3.21c.342.052.682.107 1.022.166m-1.022-.165L18.16 19.673a2.25 2.25 0 01-2.244 2.077H8.084a2.25 2.25 0 01-2.244-2.077L4.772 5.79m14.456 0a48.108 48.108 0 00-3.478-.397m-12 .562c.34-.059.68-.114 1.022-.165m0 0a48.11 48.11 0 013.478-.397m7.5 0v-.916c0-1.18-.91-2.164-2.09-2.201a51.964 51.964 0 00-3.32 0c-1.18.037-2.09 1.022-2.09 2.201v.916m7.5 0a48.667 48.667 0 00-7.5 0"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
            </button>
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
